tox.ini

```diff
@@ -11,6 +11,7 @@ deps =
     PyGObject
     pytest-cov
     pytest==3.5.1
+    ipaddress
```

Comment:
In the tox file we add only test env related dependencies, not production (or in the future, build)  ones.
It needs to be placed in the requirements.txt.
You also need to be careful on what version we depend on, as we will need nmstate in RHEL/Centos, so the relevant version should be available there.